File: Lecture 3: If-Else, While loop & Lots of Patterns/pattern1.java
import java.util.*;

class Pattern1 {
    public static void main(String[] args) {

        int n;
        System.out.print("Enter a number: ");
        Scanner sc = new Scanner(System.in);

        // taking input from user
        n = sc.nextInt();

        // closing sc to avoid unnecessary memory usage
        sc.close();

        for (int i = 1; i <= n; i++) {

            for (int j = 1; j <= n; j++) {
                System.out.print("*");
            }

            System.out.println();
        }
    }
}

File: Lecture004 Solving Pattern Questions (Part-2)/pattern1.py
class solution():
    def pattern(n):
        i=1
        while(i<=n):
            j=1
            while(j<=n):
                print(n-j+1,end=" ")
                j+=1
            print()
            i+=1

if __name__ == '__main__':
    num = int(input("Enter Number = "))
    ob=solution
    ob.pattern(num)
